### PR TITLE
Added submit button function

### PR DESCRIPTION
**Pull request by ayaan-qadri**
As issue #1 , Submit button is created and it works fine.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,8 +21,8 @@ <h2 id="word-info"></h2>
         <button id="next-btn">Next ></button>
       </div>
       <br />
-      <input type="text" />
-      <button>Submit</button>
+      <input type="text" id="wordToCheck"/>
+      <button id="validateCurrentWord">Submit</button>
```
Comment (BayLadyCoder) [inline index.html]: Can you make it snake-case like the other ids for consistency, pls?

File: index.js [hunk 2]
```diff
@@ -25,6 +25,18 @@ let currentWordIndex = 0;
 
 // todo: add shuffle button
 // todo: submit function
+
+const submitBtn = document.getElementById("validateCurrentWord");
```
Comment (BayLadyCoder) [inline index.js]: Also, update id to snake-case here.

File: index.js [hunk 3]
```diff
@@ -25,6 +25,18 @@ let currentWordIndex = 0;
...
+submitBtn.addEventListener("click", () => {
+  const wordEnterByUser = document.getElementById("wordToCheck").value.toLowerCase();
+  const currentWord = wordsToPlay[currentWordIndex].toLowerCase();
```
Comment (BayLadyCoder) [inline index.js]: Please remove `toLowerCase()` on both words because this practice is case sensitive.

File: index.js [hunk 4]
```diff
@@ -25,6 +25,18 @@ let currentWordIndex = 0;
...
+  if (currentWord == wordEnterByUser) {
+    alert("That's right spelling 🥳");
+  } else {
+    alert("That's wrong spelling,Please try again 💐");
```
Comment (BayLadyCoder) [inline index.js]: Please correct this to "That's wrong spelling. Please try again 💐".

**Review by BayLadyCoder — CHANGES_REQUESTED** [review]
Thank you so much for your contribution. You did a good job, I just have some small change requests. I will try to add more details in the issues next time, so we have clearer requirements. 👍

**Comment by BayLadyCoder** [comment]
I'd be really appreciate if you could attach some screenshots or screen recordings of the app that show how your changes work on the app.

**Comment by ayaan-qadri** [comment]
Sure, I'll make the changes and attach a screenshot tomorrow as I'm away from my laptop for today.

**Comment by ayaan-qadri** [comment]
Here is recording of working of app.

https://github.com/BayLadyCoder/spelling-practice/assets/31568025/8e77640d-0c22-460a-b57e-b2396d2082fd

**Review by BayLadyCoder — APPROVED** [review]

**Pull request merged**